.woodpecker/binaries.yml

```diff
@@ -18,6 +18,8 @@ pipeline:
           # go source code
           - "**/*.go"
           - "go.*"
+          # web source code
+          - "web/"
```

Comment:
Sorry, I think we have to adjust this to `web/**`. I tried it by adding a new test case in `pipeline/frontend/yaml/constraint/constraint_test.go` which was throwing an error:

```go
		{
			conf: "'docs/'",
			with: []string{"docs/README.md", "docs/sub/README.md", "docs/sub-sub/README.md"},
			want: true,
		},
```

Reply:
Its blob pattern ...

I'll update it ...